### DISCUSSION TITLE
Fixing the `yarn pristine` permission denied command

### DISCUSSION THREAD

**Discussion by claudealdric** [Mintbean/DevCollective.io#19] [General]
Here's what I entered to make it work for me. This allows (adds `+`) the execute permission (`x`) for the user (`u`).

```
sudo chmod u+x dev/db/init-db.sh  
```

Just enter your password, and you should be good to go. 😄

**Comment by monarchwadia** [comment]
Thanks Claude! Want to open up a PR with that change?

**Reply by claudealdric** [reply]
Where should I make the change?

**Reply by monarchwadia** [reply]
Uhh I actually already fixed it haha. Sorry.
